samples/tfm/random_hw_unique_key/prj.conf

```diff
@@ -0,0 +1,12 @@
+#
```

Comment:
The sample does not enable TFM; are you sure it needs to be under the tfm directory?

Reply:
No, I was unsure about this, but I don't know where else. @frkv @Vge0rge maybe it makes sense under `crypto/`?

Reply:
In crypto we will have the showcasing of the psa_apis, not sure if it makes sense. I would say that the root samples directory seems good to me. This is a context on it's own I think.

Reply:
I see. Maybe create a 'keys' directory, so `samples/keys/random_hw_unique_key`?

Reply:
Moved to 'keys/'.

Reply:
Fine with me